### ISSUE TITLE
No configure script in release tarball

### ISSUE THREAD

**Issue by koue** [OCL-dev/ocl-icd#16]
According to INSTALL file in the release tarball should have `configure` script. The script is missing in release tarballs.

**Comment by Kerilk** [comment]
Indeed, thanks for the report, I am not the expert on these distribution topics, Vincent is much more knowledgeable.

I think this is an oversight from when the project was completely switched to GitHub, as releases were manually packaged on the previous hosting platform. I will try to discuss the issue with Vincent to see if the best course of action is to release packages were the bootstrap script has been run, or just amend the documentation.
